src/Horse.Core.pas

```diff
@@ -16,7 +90,9 @@   THorseCore = class
     destructor Destroy; override;
     constructor Create; overload;
     class destructor UnInitialize;
+
     property Routes: THorseRouterTree read FRoutes write FRoutes;
+    function Route(APath:string):IHorseCoreRoute;
```

Comment:
In all parameters, place a space after the colon:

`function Route(APath:string):IHorseCoreRoute;` 

to

`function Route(APath: string): IHorseCoreRoute;`